test/gui/test_b0shim_tab.py

```diff
@@ -118,14 +123,21 @@ def __test_st_plugin_b0shim_dyn(view, overlayList, displayCtx, options):
             if isinstance(widget, wx.Choice) and widget.IsShown():
                 if widget.GetName() == 'optimizer-method':
                     assert set_dropdown_selection(widget, options['optimizer-method'])
-                if widget.GetName() == 'scanner-coil-order':
-                    assert set_dropdown_selection(widget, options['scanner-coil-order'])
                 if widget.GetName() == 'slices':
                     assert set_dropdown_selection(widget, options['slices'])
-                if widget.GetName() == 'scanner-coil-order':
-                    assert set_dropdown_selection(widget, options['scanner-coil-order'])
                 if widget.GetName() == 'output-value-format':
                     assert set_dropdown_selection(widget, options['output-value-format'])
+        
+        # Select the checkboxes
+        list_widgets = []
+        get_all_children(b0shim_tab.sizer_run, list_widgets)
+        for widget in list_widgets:
+            if isinstance(widget, wx.CheckBox) and widget.IsShown():
+                if widget.GetName() == 'check':
+                    widget.SetValue(True)
+                    assert set_checkbox(widget)
+                    realYield()
```

Comment:
```suggestion
                    assert set_checkbox(widget)
```

The `SetValue()` method and `realYield()` function is already being done in the `set_checkbox()` function